templates/centos-stream9.tpl.yaml

```diff
@@ -15,6 +15,16 @@ metadata:
     template.openshift.io/bindable: "false"
     template.kubevirt.io/version: v1alpha1
     defaults.template.kubevirt.io/disk: rootdisk
+{% if item.workload != "desktop" %}
+    template.kubevirt.io/containerdisks: |
+      quay.io/containerdisks/centos-stream:9
+{% if import_urls | length > 0 %}
+    template.kubevirt.io/import-urls: |
```

Comment:
I am struggeling withe the name "import-urls", even the UI shows it like this.
I wonder if `base-image` would fit better?

Reply:
`cloud-image` implies that cloud-init is used, which might be not true.

Reply:
I changed it to `image`